drivers/counter/counter_ll_stm32_rtc.c

```diff
@@ -275,7 +275,7 @@ static int rtc_stm32_init(struct device *dev)
 
 #else /* CONFIG_COUNTER_RTC_STM32_CLOCK_LSE */
 
-#ifndef(CONFIG_SOC_SERIES_STM32F4X)
+#ifndef CONFIG_SOC_SERIES_STM32F4X
```

Comment:
Should add !F1 and !F2 as well.

Reply:
Corrected.
Furthermore,  ```LL_RTC_EnableShadowRegBypass``` doesn't exist for F1 and F2 as well.
And overall the counter won't work for F2, as used ```LL_RTC_ALMA_Enable``` & ```LL_RTC_ALMA_Disable``` are not valid for F2. 
Unfortunately, I don't have F2 at hands to correct and validate F2.